static_src/js/main.js

```diff
@@ -267,7 +272,37 @@ function initializeJS() {
 	})();
 })(jQuery);
 
+// Right now this only supports a single Mermaid diagram on the page.
+var swimWithTheMermaids = function() {
+  // Are there any diagrams here?
+  if ( 1 > $( '.js-diagrams' ).length ) {
+    return undefined;
+  }
+
+  var diagrams = $( '.js-diagrams' );
+  var padding = 30;
+
+  diagrams.each( function ( idx, el ) {
+    var $el = $( el );
+    var diagramText = $( el ).find( '.js-diagrams__raw' ).text()
+    mermaid.render(
+      $el.find( '.js-diagrams__stage' ).attr( 'id' ),
+      diagramText,
+      function callbackMermaidRender( c ) {
+        $el.find( '.js-diagrams__stage' ).html( c );
+      }
+    );
+    $el.find( '.js-diagrams__stage' ).find( 'style' ).remove();
+    var updatedHeight = Math.ceil( $el.find( '.js-diagrams__stage svg .output' )[0].getBBox().height ) + padding
+    var updatedWidth = Math.ceil( $el.find( '.js-diagrams__stage svg .output' )[0].getBBox().width ) + padding
+    $el.find( '.js-diagrams__stage svg' ).height( updatedHeight );
+    $el.find( '.js-diagrams__stage' ).width( updatedWidth );
+  } );
+
+}
+
 jQuery(document).ready(function(){
     initializeJS();
     $('[rel=show-github]').showGithub();
+    swimWithTheMermaids();
```

Comment:
Sorry to be un-fun, but wdyt about calling this something more descriptive like `renderDiagrams`?